graphql/schema/rules.go

```diff
@@ -1291,6 +1291,41 @@ func lambdaDirectiveValidation(sch *ast.Schema,
 	return errs
 }
 
+func defaultDirectiveValidation(sch *ast.Schema,
```

Comment:
I think there are several other checks that need to be incorporated (thought of) here.
Like if it should work with secret, dgraph,  custom, etc directives. It is possible that we might be handling that case somewhere which I am not aware of.